src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java

```diff
@@ -116,6 +117,12 @@ public void onLoop(double timestamp)
                         break;
                     case EJECTING:
                         if (mStateChanged)
+                        {
+                            mShootTimer.reset();
```

Comment:
Whoops, I was wrong - we won't need `mShootTimer.reset()` here if we always use `mShootTimer.stop()` and `mShootTimer.reset()` in tandem